avm/res/key-vault/vault/README.md

```diff
@@ -886,6 +886,79 @@ All access policies to create.
 - Required: No
 - Type: array
 
+
+| Name | Required | Type | Description |
+| :-- | :-- | :--| :-- |
+| [`applicationId`](#parameter-accesspoliciesapplicationid) | No | string | Optional. Application ID of the client making request on behalf of a principal. |
+| [`objectId`](#parameter-accesspoliciesobjectid) | Yes | string | Required. The object ID of a user, service principal or security group in the tenant for the vault. |
+| [`permissions`](#parameter-accesspoliciespermissions) | Yes | object |  |
+| [`tenantId`](#parameter-accesspoliciestenantid) | No | string | Optional. The tenant ID that is used for authenticating requests to the key vault. |
+
+### Parameter: `accessPolicies.applicationId`
+
+Optional. Application ID of the client making request on behalf of a principal.
+
+- Required: No
+- Type: string
+
+### Parameter: `accessPolicies.objectId`
+
+Required. The object ID of a user, service principal or security group in the tenant for the vault.
+
+- Required: Yes
+- Type: string
+
+### Parameter: `accessPolicies.permissions`
+- Required: Yes
+- Type: object
+
+| Name | Required | Type | Description |
+| :-- | :-- | :--| :-- |
+| [`certificates`](#parameter-accesspoliciespermissionscertificates) | No | array | Optional. Permissions to certificates. |
+| [`keys`](#parameter-accesspoliciespermissionskeys) | No | array | Optional. Permissions to keys. |
+| [`secrets`](#parameter-accesspoliciespermissionssecrets) | No | array | Optional. Permissions to secrets. |
+| [`storage`](#parameter-accesspoliciespermissionsstorage) | No | array | Optional. Permissions to storage accounts. |
+
+### Parameter: `accessPolicies.permissions.certificates`
+
+Optional. Permissions to certificates.
+
+- Required: No
+- Type: array
+- Allowed: `[all, backup, create, delete, deleteissuers, get, getissuers, import, list, listissuers, managecontacts, manageissuers, purge, recover, restore, setissuers, update]`
+
+### Parameter: `accessPolicies.permissions.keys`
+
+Optional. Permissions to keys.
+
+- Required: No
+- Type: array
+- Allowed: `[all, backup, create, decrypt, delete, encrypt, get, getrotationpolicy, import, list, purge, recover, release, restore, rotate, setrotationpolicy, sign, unwrapKey, update, verify, wrapKey]`
+
+### Parameter: `accessPolicies.permissions.secrets`
+
+Optional. Permissions to secrets.
+
+- Required: No
+- Type: array
+- Allowed: `[all, backup, delete, get, list, purge, recover, restore, set]`
+
+### Parameter: `accessPolicies.permissions.storage`
+
+Optional. Permissions to storage accounts.
+
+- Required: No
+- Type: array
+- Allowed: `[all, backup, delete, deletesas, get, getsas, list, listsas, purge, recover, regeneratekey, restore, set, setsas, update]`
```

Comment:
Sorry if it starts to feel like bullying by know (which is not my intention), but I couldn't help but notice that your fork isn't using the latest changes to the generation script. 
We're working on catching such cases with static tests on our end, and I further suspect the utility won't change much going forward. But for the time being I'd need to ask you to
1. Fetch the lastest changes from upstream via that fancy button in your fork
1. Pull these changes to your local clone
1. Merge them back in the yout branch
1. Run the generation again

You will notice that these values will be shift into a vertical list, thought a bugfix is the more important change.

Reply:
Haha, don't worry! Will take a look 👍

Reply:
I have updated my fork with the latest changes from the upstream and ran the generation script again, but there are no new changes. Even after deleting the readme's and rerunning the generation script, there are no new changes. The Set-AVMModule function is 1-to-1 on my remote and local feature branch and the main branch.

Am I missing a step?

Reply:
Hm set PR was merged last week. Let me take a shot at that. Will update you shortly

Reply:
Shot taken and issue identified. It's an area in the inner `Set-ModuleReadMe` script that for UDTs does not format the allowed & default values as it should (that is, like it's done for top-level parameters). I'll fix this in a separate PR. Sorry for that :)